packages/calcite-components/src/controllers/usePreventDocumentScroll.browser.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import { it, expect, describe } from "vitest";
+import { mount } from "@arcgis/lumina-compiler/testing";
+import { LitElement, property } from "@arcgis/lumina";
+import { waitForAnimationFrame } from "../tests/utils/timing";
+import { usePreventDocumentScroll } from "./usePreventDocumentScroll";
+
+describe("usePreventDocumentScroll", () => {
+  class Test extends LitElement {
+    @property() opened = false;
+
+    @property() preventDocumentScroll = false;
+
+    usePreventDocumentScroll = usePreventDocumentScroll()(this);
+  }
+
+  it("should not modify document overflow when component is not opened", async () => {
+    expect(document.documentElement.style.overflow).toBe("");
```

Comment:
I think we could remove these initial assertions since tests are starting with a clean slate. If you need a comparison before interacting with the component, you could move these after `mount`.